test/e2e/multicast.go

```diff
@@ -95,7 +95,7 @@ var _ = ginkgo.Describe("Multicast", func() {
 		// Start a multicast listener on the same groups and verify it received the traffic (iperf server is the multicast listener)
 		// join multicast group (-B 224.3.3.3), UDP (-u), during (-t 30) seconds, report every (-i 1) seconds
 		ginkgo.By("creating first multicast listener pod in node " + serverNodeInfo.name)
-		iperf = fmt.Sprintf("iperf -s -B %s -u -t 30 -i 5", mcastGroup)
+		iperf = fmt.Sprintf("iperf -s -B %s -u -t 180 -i 5", mcastGroup)
```

Comment:
why are we increasing timeout? (just to see if its due to "slow" ness?) seems like test is still failing though :/ and considering it used to pass with 30 and sometimes it does... and sometimes it flakes its suspicious

Reply:
its a bug in the timing of the test. The test polls the logs of these containers for a time potentially greater than the lifecycle of the pod. So you could be polling the log of a completed pod looking for a change that will never happen. So the poll time needs to be the length of time the containers exist.

Reply:
> 3. Is this a temporary PR to catch the multicast test failures or we want to merge this in, in which case should we consider doing the same for all e2e's in the e2e.go as well?

I changed everywhere that uses a framework. I thought that would cover most of the code, but I didn't look through every test. But this is something we should merge, and grab all dbs during a control-plane failure. I hope to also figure out how to hook this into k8s e2e.